app/src/main/res/values/strings.xml

```diff
@@ -91,6 +91,7 @@
   <!-- PPO Mock Strings -->
   <string name="project_alerts_fpo">Project Alerts</string>
   <string name="alerts_fpo">Alerts (%1$s)</string>
-  <string name="address_confirmed_snackbar_text">Address confirmed! Need to change your address before it locks? Visit your backing details on our website.</string>
+  <string name="address_confirmed_snackbar_text_fpo">Address confirmed! Need to change your address before it locks? Visit your backing details on our website.</string>
```

Comment:
I snuck in this tiny string name change unrelated to this ticket because I realized I forgot to add "fpo" to the end of it